tests/passes/static-promotion/upgrade-bound.expect

```diff
@@ -21,7 +21,7 @@ static<15> component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done don
     }
   }
   control {
-    @bound(5) static repeat 5 {
+    static repeat 5 {
```

Comment:
Should the promoted control statements take i) `ir::Attributes::Default()`, or ii) the attributes of the old control statement? Maybe it should take ii): then I should prob change this before we merge.

Reply:
I think we should preserve the attributes and remove the `@promotable` (or whatever its called)

Reply:
Ok, I've done that. I also remove `@bound` once we promote to a static repeat, since it's kind of `@bound` is redundant if we have a static repeat.

Reply:
Awesome!